query-node/schema.graphql

```diff
@@ -83,23 +195,25 @@ type Channel @entity {
   # or a member. We are not handling events related to curator group so we will not set this field
   # owner: Member!
 
+  category: ChannelCategory
+
   "The title of the Channel"
-  handle: String! @fulltext(query: "search")
+  title: String! @fulltext(query: "search")
 
   "The description of a Channel"
   description: String!
 
   "Url for Channel's cover (background) photo. Recommended ratio: 16:9."
-  coverPhotoUrl: String
+  coverPhotoUrl: Asset
```

Comment:
Either the `Asset` should be `union Asset = AssetUrl | AssetStorage | Null` or this filed should be nullable.
the Url should be removed from the field name because it may not be a url.

Reply:
All the fields that come from protobuf metadata should be nullable because they are optional

Reply:
As stated above: Asset is nullable already. It can't be seen in the asset definition itself, but everywhere it is used, it is nullable (`myProperty: Asset` - without exclamation mark that marks non-nullable properties).

Reply:
I revised all content coming from protobuf and made it nullable in a couple of places where it was previously non-nullable.